pycon/migrations/0002_remove_old_google_openid_auths.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+"""
+Google OpenID auth has been turned off, so any associations that
+users had to their Google accounts via Google OpenID are now useless.
+Just remove them.
+"""
+
+from django.db import migrations
+
+
+def no_op(apps, schema_editor):
+    pass
+
+
+def remove_old_google_openid_auths(apps, schema_editor):
+    UserSocialAuth = apps.get_model('social_auth', 'UserSocialAuth')
+    db_alias = schema_editor.connection.alias
```

Comment:
On Thu, Jul 23, 2015 at 12:57 PM, Rebecca <notifications@github.com> wrote:

> > +    db_alias = schema_editor.connection.alias
>
> As far as I know, we only have one database. Is this line needed?
>

​No, just me being pedantic in Django.